### ISSUE TITLE
Layout for training data 

### ISSUE THREAD

**Issue by lhk** [rykov8/ssd_keras#77]
Thank you very much for this great port. 

I would like to train the network with a custom dataset. 
Could you add some information on the layout of the ../../frames directory and on the pickled ground truth?

How are images and labels matched to each other, what encoding is expected?

**Comment by lhk** [comment]
Oh, I'm very sorry. I've now found #15 .
Great explanations there.
Thanks
